In [ ]:
# Kwame Taylor
# Stats Quiz

In [1]:
import matplotlib.pyplot as plt
import numpy as np
from scipy import stats

In [3]:
#1
stats.poisson(61.8).sf(69)

0.16330899776626465

In [ ]:
#10

#### Assume that the time it takes to order and eat the burger special at the Esquire is normally distributed with a mean of 40 minutes and a standard deviation of 15 minutes.
 * If it takes 5 minutes to walk from Codeup to the restaurant, what is the likelihood that you can go eat at The Esquire during your lunch hour and not be late back to class?

In [14]:
np.round(stats.norm(40, 15).cdf(50), 3)

0.748

### Answers in plain text

0.163

Probability Mass Function

Inverse Survival Function

Uniform: The hottest days of the year by day of the week, because all outcomes are equally likely.
Poisson: Number of website visitors every hour, because it is measuring the number of events that occur over a given time period.
Binomial: How many people pass the BAR test for every 1000 test takers, because it is counting number of successes for every n amount of trials, in this case 1000.
Normal: Average height of horses, because it will be more likely that horses have similar heights than likely a horse is much shorter or much taller than the rest.

Uniform Distribution

Poisson Distribution

Binomial Distribution

I would use an independent t-test, with two samples: one being the ages of the web dev students, the other being the ages of the data science students.

Null hypothesis: The mean age of web dev students and the mean age of data science students are equal.

Alternative hypothesis: The mean age of web dev students is significantly lower than the mean age of data science students.

That the samples are independent of each other, the data is normally distributed, and the samples have equal variances.

t-statistic

If our t-statistic is less than zero and our p-value less than .05

If our t-statistic is equal to or more than zero and our p-value/2 is more than .05

Chi-Square Test

Null hypothesis: Data Science students and Web Dev students are equally likely to have a bachelors degree or beyond.

Alternative hypothesis: Data science students are significantly more likely to have a bachelors degree or beyond than Web Dev students.

True Positive

False Negative (Type II Error)

0.748